ideas/95-les-service-model.md

```diff
@@ -0,0 +1,78 @@
+## Preamble
+
+    Idea: #95-les-service-model
+    Title: Introduce LES Service Model
+    Status: Draft
+    Created: 2018-03-20
+
+## Summary
+
+Introduce the LES Service Model where LES clients can subscribe to higher quality services. Different
+payment models will be available, negotiation between client and full node is done via auctions based
+on demands and capacity.
+
+This way full node service provide are able to monetize the services they provide. LES will make the
```

Comment:
We have full, fast, light types of sync. When you mention `full node`, do you mean `server node` with full/fast sync mode?